src/lib/eliom_client.client.mli

```diff
@@ -315,6 +315,11 @@ val init : unit -> unit
 
 val set_reload_function : (unit -> unit -> unit Lwt.t) -> unit
 
+(** Stock the dom of the current page, which will be used when going 
+    back to this page through [window.history.back]
+*)
+val set_history_dom : unit -> unit
```

Comment:
I'd rather call this function `push_history_dom` or so. I know where the `set_` comes from but it doesn't make too much sense to me.